### ISSUE TITLE
unable to print

### ISSUE THREAD

**Issue by FlorianLudwig** [flathub/org.inkscape.Inkscape#22]
inkscape as flatpak doesn't list any of my printers

**Comment by hfiguiere** [comment]
Please try the beta build

You can install it this way:
````
flatpak remote-add flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
flatpak install flathub-beta org.inkscape.Inkscape
````

The beta build is 1.0beta1 and uses gtk3 as it is suggested above that it might work. Thanks.

**Comment by FlorianLudwig** [comment]
It looks better at least.  I do see a list of printers in that version and I can "print to pdf".  Could not test with a real printer though.

**Comment by hfiguiere** [comment]
We should be able to close this when 1.0 hits the stable branch. Not sure of the work needed to fix this on 0.92.x because of Gtk2.
